plugins/figma/README.md

```
@@ -10,15 +10,25 @@ Figma design review, diffing, and asset identification for PatternFly. Works in
 ```
 
 For enhanced results, also install the [PatternFly MCP server](https://github.com/patternfly/patternfly-mcp).
 
 ### Cursor
 
-See the [root README](../../README.md) for Cursor installation options.
+Install this plugin from the Cursor marketplace or see the [root README](../../README.md) for options.
+
+The `pf-design-mode` skill requires the official Figma plugin. Run the following in the Cursor agent chat before using that skill:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)